### DISCUSSION TITLE
github login problem

### DISCUSSION THREAD

**Discussion by nomithind** [community/community#35228] [Other Feature Feedback, Questions, & Ideas] [answered]
i have facing login problem with my github account, When i have tried to login with my normal details it says my account doesn't exist, when i try the forgotten password it says the email is not registered. The last thing i had done on the account was delete a repository, now the account is gone. so try to registered the email again, this it says the account and email do exist.
i am requested to github community, please solved my [login [problem](mudasirsajad.com).
Thanku

**Comment by airtower-luna** [comment, marked as answer]
For login issues you'll have to [contact GitHub support](https://support.github.com/request), us other users can't help there. Good luck!

**Reply by sa1dek** [reply]
I am trying to register on the Sevalal website, but it does not want to register a GetHub account.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @nomithind ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category - the Issues category is solely related to the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by lubo99** [comment]
gh auth login bad joke. I am sure I have correct token and over and over...

ocalhost:~# gh auth login

? What account do you want to log into?  [Use arrows to move, type to filter]
> GitHub.com

 What is your preferred protocol for Git operations on this host?  [Use arrows to move, type to filter]
> HTTPS
Authenticate Git with your GitHub credentials? (Y/n)

Y
How would you like to authenticate GitHub CLI?  [Use arrows to move, type to filter]
  Login with a web browser
> Paste an authentication token
Paste your authentication token: ***********************************
error connecting to api.github.com
check your internet connection or https://githubstatus.com


root@localhost:~# ping api.github.com
PING api.github.com (140.82.121.6) 56(84) bytes of data.  64 bytes from lb-140-82-121-6-fra.github.com (140.82.121.6): icmp_seq=1 ttl=50 time=32.2 ms                         64 bytes from lb-140-82-121-6-fra.github.com (140.82.121.6): icmp_seq=2 ttl=50 time=47.0 ms

token is correct page is rechable github status ok... really guys this is a bad joke... if this is the way how to complicate things to users then well done...

I would really appreciate solution! thanks

**Reply by airtower-luna** [reply]
> ? What account do you want to log into? [Use arrows to move, type to filter]
> 
> > GitHub.com

Your account appears to be `lubo99`, not `GitHub.com`. :wink:

**Reply by lubo99** [reply]
lubo99 is used not github.com lol.. github. is an offer to be selected.......lol

**Reply by lubo99** [reply]
btw i am able to login ffrom yesterday 

? What account do you want to log into? **GitHub.com** this is what you were incorrectly reffering lol
? How would you like to authenticate GitHub CLI? Paste an authentication token
Tip: you can generate a Personal Access Token here https://github.com/settings/tokens
The minimum required scopes are 'repo', 'read:org', 'workflow'.
? Paste your authentication token: ****************************************
- gh config set -h github.com git_protocol https
✓ Configured git protocol
! Authentication credentials saved in plain text
✓ Logged in as lubo99

**Comment by YinChantha** [comment]
I have a problem while login and authorize with other plateform likse vercel and other it is show me with 

You are marked as spam, and therefore cannot authorize a third party application.

Please help me in advance i really need it for now

**Comment by musabagci35** [comment]
I have been a GitHub user for approximately eight years and have several repositories associated with my account. Today, upon attempting to log in, I was directed to a different account that does not display my repositories. Additionally, the profile picture shown does not belong to my account, and it appears that my original account has disappeared.

I have verified that I am using the correct login credentials. Could you please assist me in recovering access to my original account and repositories?

**Comment by NaeemAbdullahAkram** [comment]
A simple solution to try first is to clear your browser cache and cookies. Sometimes, stored data can cause issues when logging in. After clearing them, try logging in to GitHub again, and see if it resolves the problem.
